.github/workflows/osdk_test.yml

```diff
@@ -36,7 +36,7 @@ jobs:
   # as described in the OSDK User Guide in the Asterinas Book.
   osdk-doc-env-test:
     runs-on: ubuntu-latest
-    timeout-minutes: 15
+    timeout-minutes: 30
```

Comment:
I don't think it's a proper fix. See https://github.com/asterinas/asterinas/issues/779.

Reply:
Well, I originally thought it was due to network problem, but it could indeed be due to some potential bugs. I will  leave it to an additional PR.